src/main/kotlin/com/github/quiltservertools/ledger/Ledger.kt

```diff
@@ -33,7 +33,7 @@ import net.minecraft.util.registry.Registry
 import org.apache.logging.log4j.LogManager
 import org.apache.logging.log4j.Logger
 import java.nio.file.Files
-import java.util.UUID
```

Comment:
I think our IntelliJs are configured differently, they are constantly flighting over this lol